mathesar_ui/src/component-library/text-input/TextInput.svelte

```diff
@@ -1,5 +1,6 @@
 <script lang="ts">
```

Comment:
Please rename this component differently, and revert back the name of BaseTextInput component to TextInput.

Our default input components should do no more than the basic operations.

Reply:
Conclusion: Use the following options in the fallback

1. Use InputGroup to provide the functionality of prefix and suffix
2. Create a new component wrapper for TextInput with suffix capability

Reply:
Done: 434b97

Reply:
@rajatvijay, in this PR's diff, I'm still seeing some changes in `TextInput.svelte`. I'd expect that this PR would not have any changes in that file. Those classes `'input-element'` and `'text-input'` _do_ need to be these for styling purposes. For example, if you edit a schema's name, you'll see that the name input is not styled correctly.

Reply:
@seancolsen that was a bad revert on my part. Here is the latest commit for this: 1841f1c